domain/src/main/java/com/bottlerocketstudios/brarchitecture/domain/models/Repository.kt

```diff
@@ -0,0 +1,13 @@
+package com.bottlerocketstudios.brarchitecture.domain.models
+
+import java.time.ZonedDateTime
+
+data class Repository(
```

Comment:
@VanyaHuamanBottleRocket Refactor to `GitRepository` to help prevent name clash with the interface Repository (more info in another comment)

Reply:
renaming